modules/standard/Random.chpl

```diff
@@ -89,12 +128,87 @@ record SeedGenerators {
   }
 };
 
-const SeedGenerator: SeedGenerators;
+
+// CHPLDOC FEEDBACK: If easy, I'd suggest either deprecating the 
+// :arg <type> <name>: form or else switching the order to 
+// :arg <name> <type>: as it's an easy trap to simply type the same
+// argument as in the function's signature.
+
+
+// CHPLDOC FIXME: int(64) prints out as int(64)(64)
+
+// CHPLDOC FIXME: if the first line below is shifted one character to the
+// left, it ends up being rendered like a method
+
+// CHPLDOC FIXME: if the first line below is shifted one character to the
+// right, it ends up causing a warning (promoted to error) in the .rst
+// file.  It'd be preferable to have it declare the issue in terms of
+// the .chpl line numbers.
+
```

Comment:
I think both of these case fall under invalid rst. All lines of a paragraph must be left-indented at the same level. If two lines need to have different indentation, there needs to be a line break between them. The below examples are valid because they have the empty line:

```
/*
 Fill an array...

  pseudorandom values ...
*/

// or...

/*
   Fill an array...

  pseudorandom values ...
*/
```

I added a story for improving the error messages to reference the original source line numbers.